operator/v1/types.go

```diff
@@ -243,13 +249,11 @@ type KlusterletSpec struct {
 	Namespace string `json:"namespace,omitempty"`
 
 	// RegistrationImagePullSpec represents the desired image configuration of registration agent.
-	// +required
-	// +kubebuilder:default=quay.io/open-cluster-management/registration
-	RegistrationImagePullSpec string `json:"registrationImagePullSpec"`
+	// +optional
```

Comment:
Even if we removed the `default` of the `RegistrationImagePullSpec` and `WorkImagePullSpec`, should they remain `required`?

Reply:
it is for backward compatible, I do not think we require this in the previous version

Reply:
will add a comment on what image is used if unspecified.